src/aks-preview/azext_aks_preview/agentpool_decorator.py

```diff
@@ -329,6 +364,7 @@ def init_context(self) -> None:
             self.agentpool_decorator_mode,
         )
 
+
```

Comment:
This might lead to a style issue, only one blank line is allowed be default

Reply:
thanks for pointing out